process-status-change/slack-ids.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    sgress454: 'U9KTERZ7Z',
+    'dan-greff': 'U04TEG8RM',
+    khouse: 'U6KSFH24Q',
+    craigforster: 'U9ZE7DGTE',
+    mwchapel: 'U3RKGLEG7',
+    ryanspoone: 'U93C9250Q',
+    davisonn: 'U941FUMAQ',
+    'j-m-davison': 'U04SSSA4N',
+    marcusmotill: 'U0THYQKJ6'
```

Comment:
this is a public repo probably don't want them here ... try to killl this commit if you can when removing

my suggestion is to supply a URL to the github action via the reoo config something like: `MY_SLACK_MAP = 's3.com/my_slack_map'